cmd/az-analyze/cmd/azva.go

```diff
@@ -116,20 +123,15 @@ type AzvaResourceAll struct {
 }
 
 // return azVolumeAttachements with all Pods/Nodes/Zones when no flags is provided
-func GetAllAzVolumeAttachements(namespace string) []AzvaResourceAll {
-	// access to config and Clientsets
-	config := getConfig()
-	clientsetK8s := getKubernetesClientset(config)
-	clientsetAzDisk := getAzDiskClientset(config)
-
+func GetAllAzVolumeAttachements(clientsetK8s kubernetes.Interface, clientsetAzDisk versioned.Interface, namespace string) []AzvaResourceAll {
```

Comment:
Ideally, we should've had a single resource with all the fields(nodes,zone,pod) instead of having 2 resources called AzvaResource and AzvaResourceAll. In case a user wants to get the azva by pod, node and zone would've been empty. Similarly, if node/zone is specified, pod,zone/node,pod would've been empty. But it might require some refactoring, so we can take it up later in case we've some time left at the end.